autox/service.go

```diff
@@ -0,0 +1,56 @@
+package autox
```

Comment:
Rename `service.go` to `watcher.go`. 

Remove sub folder `watcher`. Put everything under the `autox` package.